__docs__/wonder-blocks-link/link.stories.tsx

```diff
@@ -40,7 +40,7 @@ export default {
 type StoryComponentType = StoryObj<typeof Link>;
 
 /**
- * By default the link is a `primary` link.
+ * By default the link uses a blue color.
```

Comment:
Would it be helpful to reference the token name in docs instead of `blue` since it might be a different color for other themes?

Reply:
hmm great question! how about adding a semantic meaning instead?

Something in the lines of: `uses a color that communicates the presence and meaning of interaction` (extracted from the semantic colors docs).

Reply:
This sounds good!